helms/k8s-gke-service-account-assigner/values.yaml

```diff
@@ -0,0 +1,29 @@
+# Default values for k8s-gke-service-account-assigner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+defaultScopes: "https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append"
+# defaultSA: ""
+# serviceAccountName: ""
+
+image:
+  repository: imduffy15/k8s-gke-service-account-assigner
+  tag: latest
```

Comment:
we shouldn't bind to latest versions, please specify exact release version

Reply:
fixed in 088bdb1